cartography/intel/aws/wafv2.py

```diff
@@ -0,0 +1,191 @@
+import logging
+from typing import Dict
+from typing import List
+
+import boto3
+import botocore
+import neo4j
+
+from cartography.util import aws_handle_regions
+from cartography.intel.aws.ec2.util import get_botocore_config
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+@aws_handle_regions
+def get_load_waf_resources(client: botocore.client.BaseClient,
+                           elbv2_client: botocore.client.BaseClient, acl_arn: str) -> Dict[str, List[str]]:
+    resources: Dict[str, List[str]] = {'ALBs': [], 'APIGWs': []}
+    resources["APIGWs"].extend(
+        client.list_resources_for_web_acl(WebACLArn=acl_arn, ResourceType='API_GATEWAY')['ResourceArns'])
+
+    paginator = elbv2_client.get_paginator('describe_load_balancers')
+    albs = client.list_resources_for_web_acl(WebACLArn=acl_arn, ResourceType='APPLICATION_LOAD_BALANCER')[
+        'ResourceArns']
+    for page in paginator.paginate(LoadBalancerArns=albs):
+        for lb in page['LoadBalancers']:
+            resources["ALBs"].append(lb['DNSName'])
+
+    return resources
+
+
+@timeit
+def get_waf_acls(boto3_session: boto3.Session) -> List[Dict]:
+    wafv2_client = boto3_session.client('wafv2', config=get_botocore_config())
+    elbv2_client = boto3_session.client('elbv2', config=get_botocore_config())
+
+    acls: List[Dict] = []
+    for base_acl in wafv2_client.list_web_acls(Scope='REGIONAL')['WebACLs']:
+        acl = wafv2_client.get_web_acl(Name=base_acl['Name'], Scope='REGIONAL', Id=base_acl['Id'])['WebACL']
+        acl['Resources'] = get_load_waf_resources(wafv2_client, elbv2_client, acl['ARN'])
+        acls.append(acl)
+
+    return acls
+
+
+@timeit
+def load_waf_acl_alb_resources(neo4j_session, alb_ids: List[str], acl_id, current_aws_account_id, update_tag):
+    ingest_resources = """
+    UNWIND $ALB_IDS AS _ALB_ID
```

Comment:
try to use our new NodeSchema model, so that these kinds of queries are auto-generated.
https://lyft.github.io/cartography/dev/writing-intel-modules.html#load